academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Inter-Blockchain Communication Introduction"
+order: 1
+description: IBC Protocol introduction
+tag: fast-track
+---
+
+# Introduction: What is IBC?
+
+The [(Inter-Blockchain Communication (IBC)](https://ibcprotocol.org/) Protocol is _a protocol to handle authentication and transport of data between two blockchains_. IBC **requires a minimal set of functions**, specified in the [Interchain Standards (ICS)](https://github.com/cosmos/ibc/tree/master/spec/ics-001-ics-standard). Notice that those specifications do not limit the network topology or consensus algorithm, so IBC can be used with a wide range of blockchains or state machines. IBC is intended to work in execution environments where participants do not necessarily trust each other.
```

Comment:
Not sure about the last sentence here: _IBC is intended to work in execution environments where participants do not necessarily trust each other._

Instead, I would suggest focussing on what IBC adds specifically, so something like: _The IBC protocol provides a **permissionless** way for relaying data packets between blockchains, unlike most trusted bridging technologies. The security of IBC reduces to the security of the participating chains._